src/sensai/data_transformation.py

```diff
@@ -224,17 +337,26 @@ def apply(self, df: pd.DataFrame):
                 df["%s_%d" % (columnName, i)] = encodedArray[:, i]
         return df
 
+    def summary(self):
+        summary = super().summary()
+        summary["columns"] = self.columns
+        summary["inplace"] = self.inplace
+        summary["handleUnknown"] = self.handleUnknown
+        summary["inferCategories"] = self.inferCategories
+        return summary
 
+
+# TODO: we have a feature generator doing the same thing. Can we remove this and its child below?
```

Comment:
No. Having a feature generator with the same functionality does not make these transformers obsolete. DFTs are used for a different purpose, and in particular, are chainable, whereas FeatureGenerators are not.
Also: compatibility is more important than reduced redundancy.

Reply:
Same question as above, how about we leave the TODO?

While I agree that it is a bit contrived, ChainedFeatureGenerator is a pretty useful concept in my opinion. At least I tend to use it quite a lot

Reply:
We shouldn't leave the TODO, because the point is that DFTs are useful in their own right (first point I made above). Someone might use DFTs in a completely different context (where FGs are not a thing) and then might need this functionality.

Reply:
True that, removing it

Reply:
Done